packages/unity-js-bridge/src/thirdweb-bridge.ts

```diff
@@ -196,6 +208,15 @@ class ThirdwebBridge implements TWBridge {
       }
     }
 
+    if (addrOrSDK.startsWith("auth")) {
```

Comment:
you need to do the change in unity in `Wallet.cs` in the `Authenticate()` method, switch from sdk/auth/login to just auth/login